torchaudio/backend/_soundfile_backend.py

```diff
@@ -210,6 +210,10 @@ def load_wav(
 ) -> Tuple[torch.Tensor, int]:
     """Load wave file.
 
+    Warnings:
+        This functions is being deprecated and will be removed in ``0.9.0``.
```

Comment:
"This function is deprecated and will be removed in the 0.9.0 release. Use load(..., normalize=False), instead."